src/ideation/ideation.service.ts

```diff
@@ -1,20 +1,31 @@
 import { Injectable } from "@nestjs/common";
 import { PrismaService } from "src/prisma/prisma.service";
-// import { CreateIdeationDto } from "./dto/create-ideation.dto";
-// import { UpdateIdeationDto } from "./dto/update-ideation.dto";
+import { CreateIdeationDto } from "./dto/create-ideation.dto";
+import { UpdateIdeationDto } from "./dto/update-ideation.dto";
 
-// const USER_ID = "bf24212d-403f-4459-aa76-d9abc701a3bf";
+//const USER_ID = "bf24212d-403f-4459-aa76-d9abc701a3bf";
+//const UserId = 10;
 
 @Injectable()
 export class IdeationService {
     constructor(private prisma: PrismaService) {}
-    // create(createIdeationDto: CreateIdeationDto) {
-    //     return "This action adds a new ideation";
-    // }
+    async create(createIdeationDto: CreateIdeationDto) {
+        const { title, description, vision } = createIdeationDto;
 
-    // findAll() {
-    //     return "This action returns all ideation";
-    // }
+        const createdIdeation = await this.prisma.projectIdea.create({
+            data: {
+                userId: 3,
```

Comment:
userId shouldn't be hardcoded. everytime the data is reseeded you get a different ID for everything